src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -790,7 +797,9 @@ const Prescriptions = () => {
       {content()}
       <PrescriptionsPrintOnly
         list={printedList}
-        hasError={hasFullListDownloadError || isAlertVisible || allergiesError}
```

Comment:
Looks like allergiesError may not always be a boolean. Suggested change:
Inline:
<PrescriptionsPrintOnly
  list={printedList}
  hasError={hasFullListDownloadError || isAlertVisible || !!allergiesError}
  isFullList={printedList.length === prescriptionsFullList.length}
/>

Or, extracted for readability:

const hasPrintError =
  hasFullListDownloadError || isAlertVisible || Boolean(allergiesError);

<PrescriptionsPrintOnly
  list={printedList}
  hasError={hasPrintError}
  isFullList={printedList.length === prescriptionsFullList.length}
/>